### PR TITLE
DOC-4272 Add support for versioned products

### PR DESCRIPTION
**Pull request by paoloredis**
These changes enable version dropdown for Kubernetes and RS, provided that their linkTitles are respectively `Redis for Kubernetes` and `Redis Software`.

Preview link: https://redis.io/docs/staging/DOC-4272/operate/ . The dropdowns are not shown since the linkTitles are not correct yet

### CODE REVIEW

File: layouts/partials/docs-nav.html [hunk 1]
```diff
@@ -17,7 +17,31 @@
...
       </a>
+      {{if (eq (.Params.linkTitle) "Redis for Kubernetes")}}
+      <div id="versionSelectorKubernetes" class="menu__version-selector version-selector-control" onclick="_openVersionSelector('Kubernetes')" style="display: none;">
+        <button class="menu__version-selector-btn version-selector-control">
+            <span id="versionSelectorKubernetesValue" class="version-selector-control">latest</span>
+            <span class="menu__version-selector__toggler opener version-selector-control">&#x25BC;</span>
+            <span class="menu__version-selector__toggler closer version-selector-control">&#x25B2;</span>
+        </button>
+        <div id="versionDropdownKubernetes" class="menu__version-selector__list version-selector-control">
+            <a href="https://dev.learn.redis.com/docs/latest/operate/kubernetes/" id="kubernetes-version-select-latest" onclick="_setSelectedVersion('kubernetes', 'latest')">latest</a>
```
Comment (rrelledge) [inline layouts/partials/docs-nav.html:28]: Do the links need to be changed from `dev.learn.redis.com` to `redis.io`? Or is this just for testing/staging purposes?
```suggestion
            <a href="https://redis.io/docs/latest/operate/kubernetes/" id="kubernetes-version-select-latest" onclick="_setSelectedVersion('kubernetes', 'latest')">latest</a>
```

File: layouts/partials/docs-nav.html [hunk 2]
```diff
@@ -17,7 +17,31 @@
...
+      </div>
+      {{else if (eq (.Params.linkTitle) "Redis Software")}}
+      <div id="versionSelectorRs" class="menu__version-selector version-selector-control" onclick="_openVersionSelector('Rs')" style="display: none;">
+        <button class="menu__version-selector-btn version-selector-control">
+            <span id="versionSelectorRsValue" class="version-selector-control">latest</span>
+            <span class="menu__version-selector__toggler opener version-selector-control">&#x25BC;</span>
+            <span class="menu__version-selector__toggler closer version-selector-control">&#x25B2;</span>
+        </button>
+        <div id="versionDropdownRs" class="menu__version-selector__list version-selector-control">
+            <a href="https://dev.learn.redis.com/docs/latest/operate/rs/" id="rs-version-select-latest" onclick="_setSelectedVersion('rs', 'latest')">latest</a>
```
Comment (rrelledge) [inline layouts/partials/docs-nav.html:39]: Same question here
```suggestion
            <a href="https://redis.io/docs/latest/operate/rs/" id="rs-version-select-latest" onclick="_setSelectedVersion('rs', 'latest')">latest</a>
```

**Review by rrelledge — APPROVED** [review]
Thank you! Looks good to me aside from my question about the base URL links.

File: layouts/partials/docs-nav.html (hunk 1 above)
Comment (paoloredis) [reply layouts/partials/docs-nav.html]: Well spotted, thank you

**Pull request merged**